### PR TITLE
Add debugpy auto start with `FLYTE_INTERNAL_START_DEBUGPY`

### PR DESCRIPTION
**Pull request by ggydush**
# TL;DR
Adds debugpy auto-start adapter to `setup_execution` so follow-up registration is not required to debug.

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: flytekit/bin/entrypoint.py [hunk 1]
```diff
@@ -280,6 +280,13 @@ def setup_execution(
             )
         cb = cb.with_serialization_settings(ssb.build())
 
+    debugger_on = get_one_of("FLYTE_INTERNAL_START_DEBUGPY")
```
Comment (kumare3) [inline flytekit/bin/entrypoint.py:283]: can we make this Env var little smaller, we have been going towards the following convention
`__FF_DBGPY`

File: flytekit/core/python_function_task.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def __init__(
         if task_function is None:
             raise ValueError("TaskFunction is a required parameter for PythonFunctionTask")
         self._native_interface = transform_function_to_interface(task_function, Docstring(callable_=task_function))
-        mutated_interface = self._native_interface.remove_inputs(ignore_input_vars)
+    mutated_interface = self._native_interface.remove_inputs(ignore_input_vars)
```
Comment (kumare3) [inline flytekit/core/python_function_task.py:119]: seems like a wrong indentation

File: flytekit/bin/entrypoint.py (hunk 1 above)
Comment (ggydush) [reply flytekit/bin/entrypoint.py]: Added `__FF_DBGPY` and `__FF_DBGPY_PORT`

File: flytekit/core/python_function_task.py (hunk 2 above)
Comment (ggydush) [reply flytekit/core/python_function_task.py]: Accidental, fixed!

**Pull request closed without merging**